_docs/project-files/xml-format.md

```diff
@@ -27,49 +27,49 @@ First, every node in the project file format supports `if` and `unless` attribut
  * ios, android, windows, mac, linux or html5
  * cpp, neko, flash or js
 
-You can `<set />` or `<unset />` values for conditional logic:
```

Comment:
Even if it's not right here, I think the section for set should still mention unset.

Similarly, the define section should probably mention undefine.

Reply:
Oops! I realized this is the general conditionals section, and not specifically for set. Perhaps just a casual mention of unset and undefine here is good enough.

Reply:
They actually weren't mentioned in the `<set />` and `<define />` sections, so I added that too. How's it look now?

Reply:
Looks good!